### ISSUE TITLE
`math.randomseed` has compatibility issue with certain apps

### ISSUE THREAD

**Issue by nadecancode** [dragonflydb/dragonfly#3919]
**Describe the bug**
It seems like the Lua version that Dragonfly has upgraded to does not take floats for `math.randomseed`, which the older versions of Lua have probably accounted for that behavior. This results Sentry's incompatibility with Dragonfly. @aarnaud has made a PR to Sentry https://github.com/getsentry/sentry/pull/65825/files but it seems like Sentry just let it stale without addressing the problem. It would be nice if Dragonfly can address this, since I suppose this should be a part of "full Redis compatibility".

**To Reproduce**
Use `math.randomseed` with a float

**Expected behavior**
`math.randomseed` correctly converts float to integer internally and returns a seed

**Screenshots**

**Reproducible Code Snippet**
https://github.com/getsentry/sentry/pull/65825/files#diff-b8fbfcdbe3cc5f51bb7b01328639cea8b1180a798e18ac03f3a5293e216bf47fL357

**Additional context**

**Comment by romange** [comment]
@dranikpg  can you please take a look? Is it possible to inject our own implementation of `math.randomseed` ?

**Comment by dranikpg** [comment]
Yes, we can inject a custom implementation, we can also finally add support for redis log 🙂

**Comment by romange** [comment]
closing as it was merged into sentry
